### DISCUSSION TITLE
Error running hello-triagle compiled to wasm

### DISCUSSION THREAD

**Discussion by TheMrAmazing** [gfx-rs/wgpu#1806] [Q&A]
I am having trouble running the hello-triangle compiling to wasm. It has compiled fine, and I am able to run this example: https://wgpu.rs/examples/?example=hello-triangle but when I try and run that same sample compiled myself on the same browser, I get these errors:
![image](https://user-images.githubusercontent.com/64285083/129492453-563c3f04-0c41-49ad-966e-5d0b94af401f.png)

I am just starting out with Rust, WASM, and WebGPU so I may be making a dumb mistake. Let me know if you need any more information.

Thanks

**Comment by grovesNL** [comment]
Sometimes the web backend and browsers (Firefox Nighlty and Chrome Canary) fall out of sync (the web API still changes occasionally) and we'll get errors like this. We try to sync up the web backend every so often, but in the meantime you could use an older version of wgpu or the browsers.
